Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -94,13 +94,21 @@ public function publishCommand(string $collection = null)
             }
 
             foreach ($collections as $collection) {
+                $progressIndicator = new ProgressIndicator($this->output->getOutput());
                 try {
                     /** @var CollectionInterface $collection */
                     $this->outputLine('Publishing resources of collection "%s"', [$collection->getName()]);
+                    $progressIndicator->start('Published 0');
```

Comment:
If this is used in automated ways, the output might be unexpected. What about adding a `quiet` flag, so it can at least be disabled?

Reply:
sure. I think @robertlemke was thinking about generic `--quiet` and `--no-input` for a while ... but for this case we could start with an explicit `$quiet` parameter

Reply:
adjusted